toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -74,8 +75,8 @@ public final void test(final String name, final String idProject, final boolean
 		DAOManager.saveTestPlan(testPlanTemplate);
 	}
 
-	public final void testAndStore(String apiKey, final ITestPlan project) throws Exception {
-		testAndStore(apiKey, project, false);
+	public final void testAndStore(String apiKey, final ITestPlan testplan) throws Exception {
+		testAndStore(apiKey, testplan, false);
 	}
 
 	public final void testAndStore(String apiKey, final ITestPlan testPlan, final boolean useRemoteRepository) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)